python-sdk/pyproject.toml

```diff
@@ -56,9 +56,11 @@ tests = [
 ]
 google = [
     "protobuf<=3.20", # Google bigquery client require protobuf <= 3.20.0. We can remove the limitation when this limitation is removed
-    "apache-airflow-providers-google>=6.4.0",
+    "apache-airflow-providers-google>6.4.0",
```

Comment:
```suggestion
    "apache-airflow-providers-google==8.8.0",
```

Reply:
@utkarsharma2 as we discussed, we should try to have 8.8.x as an upper threshold - and remove the other pins. If this works, I suggest we report the issue and suggest to the Google provider team to pin the other dependencies...!

It does feel that the issue is within the Google provider's dependencies - and it would be better if we addressed the versions of 
```
    "pandas-gbq>=0.15.0",
    "protobuf>2.5.0"
```

Reply:
Tried this, didn't work, still installing pandas-gbq==0.13.2 as there is no min dependency on the provider https://pypi.org/project/apache-airflow-providers-google/8.8.0/